aws-xray-propagator/src/main/java/io/opentelemetry/contrib/awsxray/propagator/AwsXrayLambdaConfigurablePropagator.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.contrib.awsxray.propagator;
+
+import io.opentelemetry.context.propagation.TextMapPropagator;
+import io.opentelemetry.sdk.autoconfigure.spi.ConfigProperties;
+import io.opentelemetry.sdk.autoconfigure.spi.ConfigurablePropagatorProvider;
+
+/**
+ * A {@link ConfigurablePropagatorProvider} which allows enabling the {@link
+ * AwsXrayLambdaPropagator} with the propagator name {@code xray-lambda}.
+ */
+public final class AwsXrayLambdaConfigurablePropagator implements ConfigurablePropagatorProvider {
+  @Override
+  public TextMapPropagator getPropagator(ConfigProperties config) {
+    return AwsXrayLambdaPropagator.getInstance();
+  }
+
+  @Override
+  public String getName() {
+    return "xray-lambda";
```

Comment:
Seems it is from spec, we may want to update spec to be `awslambda` or `aws-lambda`.

Reply:
If we implement support for w3c like I suggested above, using your naming suggestion would not allow a distinction between tracecontext-lambda vs tracecontext, and would also not allow users to choose just xray or just tracecontext.

Reply:
Correct me if I am wrong, the current solution for w3c in Lambda works for API GW -> Lambda case, only if the http request has w3c trace headers?